### PR TITLE
Disable update check in the CI. Add User-Agent in http.Client{}

### PR DESCRIPTION
**Pull request by guillaumerose**
It will help counting the number of clusters started.

### CODE REVIEW

File: pkg/crc/version/version.go [hunk 1]
```diff
@@ -87,7 +87,12 @@ func GetCRCLatestVersionFromMirror() (*CrcReleaseInfo, error) {
 	client := &http.Client{
 		Timeout: 5 * time.Second,
 	}
-	response, err := client.Get(releaseInfoLink)
+	req, err := http.NewRequest(http.MethodGet, releaseInfoLink, nil)
+	if err != nil {
+		return nil, err
+	}
+	req.Header.Set("User-Agent", fmt.Sprintf("crc/%s", crcVersion))
```
Comment (praveenkumar) [inline pkg/crc/version/version.go:94]: @guillaumerose this is only added to understand for which version the request made?

Comment (guillaumerose) [reply pkg/crc/version/version.go:94]: yes, it is also a good practice to specify a proper user agent. I should also add the platform.

File: pkg/crc/version/version.go [hunk 2]
```diff
@@ -87,7 +87,12 @@ func GetCRCLatestVersionFromMirror() (*CrcReleaseInfo, error) {
 	client := &http.Client{
 		Timeout: 5 * time.Second,
 	}
-	response, err := client.Get(releaseInfoLink)
+	req, err := http.NewRequest(http.MethodGet, releaseInfoLink, nil)
```
Comment (anjannath) [inline pkg/crc/version/version.go:90]: Don't we need to use the `CRC_DISABLE_UPDATE_CHECK` env variable to stop this check?

Comment (guillaumerose) [reply pkg/crc/version/version.go:90]: The variable is automatically bound to the config key via that config package. It is a magic feature of viper.

**Pull request merged**